services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -354,7 +353,24 @@ async def remove_orphaned_services(
             not await is_node_id_present_in_any_project_workbench(app, node_id)
             or node_id not in currently_opened_projects_node_ids
         ):
-            logger.info("Will remove service %s", interactive_service["service_host"])
+            service_host = interactive_service["service_host"]
+            if interactive_service.get("service_state") == "pulling":
```

Comment:
maybe there is more the one state which should be monitored here? Like "created" all the ones before "running". I would also add those.

Reply:
The states are NOT one-to-one with those of docker but as defined [in the director](https://github.com/ITISFoundation/osparc-simcore/blob/0825cff7ef203a831a11fc8db0a4ac19ae8aad60/services/director/src/simcore_service_director/producer.py#L514) service